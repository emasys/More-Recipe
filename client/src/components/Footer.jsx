import React from 'react';


const Footer = () => (
  <div>
    <footer className="container-fluid footer-main bg-dark">
      <div className="container ">
        <div className="row ">
          <div className="col-12 footer-social text-center mt-20">
            <a href="https://web.facebook.com/emasys23" rel="noopener noreferrer" target="_blank">
              <i className="fa fa-facebook-square text-white fa-2x" aria-hidden="true" />
            </a>&nbsp;
            <a href="https://twitter.com/emasys_nd" rel="noopener noreferrer" target="_blank">
              <i className="fa fa-twitter-square text-white fa-2x" aria-hidden="true" />
            </a>&nbsp;
            <a href="https://github.com/emasys" rel="noopener noreferrer" target="_blank">
              <i className="fa fa-github-square text-white fa-2x" aria-hidden="true" />
            </a>
            <p className="text-center text-white">
              &copy; {new Date().getFullYear()}
            </p>
          </div>
        </div>
      </div>
    </footer>
  </div>
);

export default Footer;
