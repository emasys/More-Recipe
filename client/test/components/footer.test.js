import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../src/components/Footer';

test('should render Footer correctly', () => {
  const wrapper = shallow(<Footer/>);
  expect(wrapper).toMatchSnapshot();
});
