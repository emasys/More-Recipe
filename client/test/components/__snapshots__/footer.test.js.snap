// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Footer correctly 1`] = `
<div>
  <footer
    className="container-fluid footer-main bg-dark"
  >
    <div
      className="container "
    >
      <div
        className="row "
      >
        <div
          className="col-12 footer-social text-center mt-20"
        >
          <a
            href="https://web.facebook.com/emasys23"
            rel="noopener noreferrer"
            target="_blank"
          >
            <i
              aria-hidden="true"
              className="fa fa-facebook-square text-white fa-2x"
            />
          </a>
           
          <a
            href="https://twitter.com/emasys_nd"
            rel="noopener noreferrer"
            target="_blank"
          >
            <i
              aria-hidden="true"
              className="fa fa-twitter-square text-white fa-2x"
            />
          </a>
           
          <a
            href="https://github.com/emasys"
            rel="noopener noreferrer"
            target="_blank"
          >
            <i
              aria-hidden="true"
              className="fa fa-github-square text-white fa-2x"
            />
          </a>
          <p
            className="text-center text-white"
          >
            © 
            2018
          </p>
        </div>
      </div>
    </div>
  </footer>
</div>
`;
